plugins/modules/proxysql_query_rules.py

```diff
@@ -75,6 +75,13 @@
         operator in front of the regular expression matching against
         match_pattern.
     type: bool
+  re_modifiers:
+    description:
+      - Comma separated list of options to modify the behavior of the RE engine.
+        With C(CASELESS) the match is case insensitive. With C(GLOBAL) the replace
+        is global (replaces all matches and not just the first).
+        For backward compatibility, only C(CASELESS) is the enabled by default.
+    version_added: "2.9"
```

Comment:
version added would be `1.3.0`.  
can you also please add a changelog fragement @ls-michielrensen?

futhermore I'm afraid that this will break backwards compatibility with proxysql 1.x.x like #71 #73

Reply:
Yeah it probably is not backwards compatible no, do we need to add a version constraint somewhere?

Reply:
basically you can adopt like this: https://github.com/ansible-collections/community.proxysql/pull/73/commits/ca47c3218adbefbc8fd150e8e93d50cf71e3915c  
This is imo sufficient. The integration test must be updated/prepared. I'm working on that already in #73

Reply:
link to the changelog related doc for convenience https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment